modules/ROOT/pages/syntax-quick-reference.adoc

```diff
@@ -0,0 +1,1290 @@
+= AsciiDoc Syntax Quick Reference
+:navtitle: Syntax Quick Reference
+:description: The quick reference for common AsciiDoc document and text formatting markup.
+:collapsible:
+:url-char-xml: https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references
+:url-data-uri: https://developer.mozilla.org/en-US/docs/data_URIs
+
+[IMPORTANT]
+The examples on this page focus on the output produced for the html backend.
+An AsciiDoc converter is expected to produce complementary output when generating other output formats, such as PDF, EPUB, and DocBook.
+
+== Paragraphs
+
+.Paragraph
+[#ex-normal]
+----
+include::text:example$text.adoc[tag=b-para]
+----
+
+.View result of <<ex-normal>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=b-para]
+====
+
+.Literal paragraph
+[#ex-literal]
+----
+include::verbatim:example$literal.adoc[tag=qr-para]
+----
+
+.View result of <<ex-literal>>
+[%collapsible.result]
+====
+include::verbatim:example$literal.adoc[tag=qr-para]
+====
+
+.Hard line breaks
+[#ex-hardbreaks]
+----
+include::text:example$text.adoc[tag=hb-all]
+----
+
+.View result of <<ex-hardbreaks>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=hb-all]
+====
+
+.Lead paragraph
+[#ex-lead]
+----
+include::text:example$text.adoc[tag=qr-lead]
+----
+
+.View result of <<ex-lead>>
+[%collapsible.result]
+====
+include::text:example$text.adoc[tag=qr-lead]
+====
+
+TIP: The default Asciidoctor stylesheet automatically styles the first paragraph of the preamble as a xref:blocks:preamble-and-lead.adoc[lead paragraph] if no role is specified on that paragraph.
+
+== Text formatting
+
+.Bold, italic, and monospace
+[#ex-format]
+----
+include::text:example$text.adoc[tag=b-bold-italic-mono]
```

Comment:
suggestion: I think what we did in the `--help syntax` is more helpful:

````
.Constrained (applied at word boundaries)
[#ex-format]
----
*strong importance* (aka bold)

_stress emphasis_ (aka italic)

`monospaced` (aka typewriter text)

*_strong with a bit of emphasis_*

cannot pick one, let's use them `*_all_*`
----
````

````
.Unconstrained (applied anywhere)
[#ex-format]
----
**C**reate+**R**ead+**U**pdate+**D**elete

fan__freakin__tastic

``mono``culture

Beatle**__mania__**
----
````

1. it gives the general rule: 
    * Constrained -> applied at word boundaries
    * Unconstrained -> applied anywhere
2. it gives a bit of context for each examples
3. it uses "real-life" examples

Repeating the words "constrained" and "unconstrained" is a bit boring no? :smile:

Reply:
I'd be open to changing. I wrote this before we came up with the examples in the syntax help, so more creativity happened.

Reply:
> suggestion: I think what we did in the --help syntax is more helpful:
> Repeating the words "constrained" and "unconstrained" is a bit boring no?

:+1: